core/src/com/unciv/ui/overviewscreen/CityOverviewTable.kt

```diff
@@ -14,16 +17,22 @@ import kotlin.math.roundToInt
 
 class CityOverviewTable(val viewingPlayer: CivilizationInfo, val overviewScreen: EmpireOverviewScreen): Table() {
 
-    val columnsNames = arrayListOf("Population", "Food", "Gold", "Science", "Production", "Culture", "Happiness")
+    companion object {
```

Comment:
Why in a companion object though... what do you gain from this?

Reply:
Allows const keyword?

Reply:
I count roughly nine classes in Unciv that already have companion objects containing nothing but constants or constant-for-the-whole-app-run values. So where does in-code documentation to express "this `8` is a number of columns" go as best practice? I'd like to do this by declaring it as a constant with a descriptive name and somehow collect those for readability near the top of their file...

 I'll gladly help working towards a consistent best-practice coding style or documenting it...

Reply:
Const val isn't possible from within the class? O_O
Huh